app/javascript/react/screens/App/Overview/screens/MappingWizard/MappingWizard.js

```diff
@@ -70,76 +44,55 @@ class MappingWizard extends React.Component {
+        render: () => <MappingWizardGeneralStep />,
+        reduxFormKey: 'mappingWizardGeneralStep'
+      },
+      {
+        title: __('Clusters'),
+        render: () => this.mappingWizardClustersStepContainer,
+        reduxFormKey: 'mappingWizardClustersStep'
+      },
+      {
+        title: __('Datastores'),
+        render: () => this.mappingWizardDatastoresStepContainer,
+        reduxFormKey: 'mappingWizardDatastoresStep'
+      },
+      {
+        title: __('Networks'),
+        render: () => this.mappingWizardNetworksStepContainer,
+        reduxFormKey: 'mappingWizardNetworksStep',
+        onNext: this.createMappings
+      },
+      {
+        title: __('Results'),
+        render: () => this.mappingWizardResultsStepContainer,
+        reduxFormKey: 'mappingWizardResultsStep'
+      }
+    ];
 
     return (
-      <Modal
-        show={!hideMappingWizard}
+      <ModalWizard
+        numSteps={5}
+        showWizard={!hideMappingWizard}
         onHide={hideMappingWizardAction}
         onExited={mappingWizardExitedAction}
-        dialogClassName="modal-lg wizard-pf"
-      >
-        <Wizard>
-          <Modal.Header>
-            <button
-              className="close"
-              onClick={hideMappingWizardAction}
-              aria-hidden="true"
-              aria-label="Close"
-            >
-              <Icon type="pf" name="close" />
-            </button>
-            <Modal.Title>{__('Infrastructure Mapping Wizard')}</Modal.Title>
-          </Modal.Header>
-          <Modal.Body className="wizard-pf-body clearfix">
-            <MappingWizardBody
-              loaded
-              activeStepIndex={activeStepIndex}
-              activeStep={activeStep}
-              goToStep={this.goToStep}
-              disableNextStep={disableNextStep}
-              transformationsBody={transformationsBody}
-            />
-          </Modal.Body>
-          <Modal.Footer className="wizard-pf-footer">
-            <Button
-              bsStyle="default"
-              className="btn-cancel"
-              onClick={hideMappingWizardAction}
-            >
-              {__('Cancel')}
-            </Button>
-            <Button
-              bsStyle="default"
-              onClick={this.prevStep}
-              disabled={onFirstStep}
-            >
-              <Icon type="fa" name="angle-left" />
-              {__('Back')}
-            </Button>
-            <Button
-              bsStyle="primary"
-              onClick={onFinalStep ? hideMappingWizardAction : this.nextStep}
-              disabled={disableNextStep}
-            >
-              {onFinalStep
-                ? __('Close')
-                : activeStepIndex === 3 ? __('Create') : __('Next')}
-              <Icon type="fa" name="angle-right" />
-            </Button>
-          </Modal.Footer>
-        </Wizard>
-      </Modal>
+        title={__('Infrastructure Mapping Wizard')}
+        shouldDisableNextStep={activeStepIndex => {
```

Comment:
we now need to ensure this piece will work w/ our newest validations in Mapping Wizard (since this PR is conflating that piece too)... @michaelkro can you validate this?

Reply:
I didn't touch the validation code, and wired up the same form data to the same place in theory... Definitely should "just work", fingers crossed!

Reply:
I didn't touch the validation code, and wired up the same form data to the same place in theory... Definitely should "just work", fingers crossed!